app/views/api_docs/reference/reference.html.erb

```diff
@@ -29,8 +29,7 @@
 <% @document.paths.each do |path_name, path| %>
   <% %w[get put post delete patch].map do |http_verb| %>
     <% operation = path.node_data[http_verb]
-    next unless operation && !operation.kind_of?(Openapi3Parser::Node::Placeholder)
```

Comment:
All of these should be solved by https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/827.